apm-agent-attach-cli/src/main/java/co/elastic/apm/attach/UserRegistry.java

```diff
@@ -222,5 +249,111 @@ public boolean isCurrentUser() {
         public String getUsername() {
             return username;
         }
+
+        public static CommandOutput executeCommand(List<String> command) {
+            ProcessBuilder buildTheProcess = new ProcessBuilder(command);
+            // merge stdout and stderr so we only have to read one stream
+            buildTheProcess.redirectErrorStream(true);
+            Process spawnedProcess = null;
+            int exitValue = -1;
+            Exception exception = null;
+            StringBuilder commandOutput = new StringBuilder();
+            try {
+                spawnedProcess = buildTheProcess.start();
+
+                long timeout = 5 * 1000L; // NOTE 5 second timeout!
+                long start = System.currentTimeMillis();
+                long now = start;
+                boolean isAlive = true;
+                byte[] buffer = new byte[64 * 1000];
```

Comment:
[minor] quite a big buffer for command output, but since this is a standalone JVM it's that big of a deal.

Reply:
Sorry, not sure whether you're saying it matters or doesn't matter. This would be called in the agent not the spawned process, but the buffer would only exist during the call. I can reduce it with no real issue though. What is your recommendation, 8k or 1k?

Reply:
Shoot, I missed an important word in this sentence - its _**NOT**_ a big of a deal 🤦‍♂️ 
By standalone JVM I meant the cli tool rather than the user's JVM. I was only referring to the allocation of the buffer and it's really not super important.

Generally speaking, [minor] comments are used to say - "I noticed something, you may want to reconsider, but ignoring is perfectly acceptable". In this case, it was only about the fact that it's more than I'd expect for this purpose.

Reply:
thanks, changed the buffer to 4k in commit b0e8e4cf1c1db67e3cdb2aaedf7e2c26a331d902